pypit/arsciexp.py

```diff
@@ -696,29 +609,23 @@ def MasterWaveCalib(self, fitsdict, sc, det):
         if self._wvcalib[det-1] is not None:
             msgs.info("An identical master wave calib frame already exists")
             return False
-        if self._argflag['reduce']['usewave'] in ['wave']:
-            # Attempt to load the Master Frame
-            if self._argflag['masters']['use']:
-                mswv_calib_name = armasters.master_name(self._argflag['run']['masterdir'],
-                                                       'wave_calib', self._argflag['masters']['setup'])
-                try:
-                    wv_calib = arload.load_master(mswv_calib_name, frametype="wv_calib")
-                except IOError:
-                    msgs.warn("No MasterWave1D data found {:s}".format(mswv_calib_name))
-                else:
-                    self._argflag['masters']['loaded'].append('wave_calib'+self._argflag['masters']['setup'])
-            if 'wave_calib'+self._argflag['masters']['setup'] not in self._argflag['masters']['loaded']:
-                # Setup arc parameters (e.g. linelist)
-                arcparam = ararc.setup_param(self, sc, det, fitsdict)
-                self.SetFrame(self._arcparam, arcparam, det)
-                ###############
-                # Extract arc and identify lines
-                wv_calib = ararc.simple_calib(self, det)
-                #
-        else: # It must be the name of a file the user wishes to load
-            msgs.error("Not prepared to read a user-specified wave file")
-            #mswv_calib_name = self._argflag['run']['masterdir']+'/'+self._argflag['reduce']['usewavecalib']
-            #wv_calib = arload.load_master(mswv_calib_name, frametype="wv_calib")
+        # Attempt to load the Master Frame
```

Comment:
I feel we have to make this series of methods related to MasterFrames (and more) 
easier to read (and write).  But now is not the time to do it.